src/js/messaging/routes.jsx

```diff
@@ -1,8 +1,34 @@
-import AppMessaging from './containers/AppMessaging';
+import MessagingApp from './containers/MessagingApp';
+import Folder from './containers/Folder';
+import Main from './containers/Main';
 
 const routes = {
   path: '/messaging',
-  component: AppMessaging
+  component: MessagingApp,
+  indexRoute: {
+    onEnter: (nextState, replace) => replace('/messaging/folder/0')
+  },
+  childRoutes: [
+    {
+      path: '',
+      component: Main,
+      childRoutes: [
+        { path: 'folder/:id', component: Folder }
```

Comment:
Would like to alias these with more user friendly URLs, at least for the main folders, while still passing the folder id parameter. Something like:

Inbox = `/messaging/inbox`
Drafts = `/messaging/drafts`
Sent = `/messaging/sent`
Deleted = `/messaging/deleted`

Maybe we can map "aliases" (perhaps folder name in lowercase) to the ids somewhere in the store.